packages/fiber/src/core/renderer.ts

```diff
@@ -376,7 +376,6 @@ function createRenderer<TCanvas>(roots: Map<TCanvas, Root>) {
         array.splice(array.indexOf(beforeChild), 0, child)
       } else if (child.attachObject || (child.attach && !is.fun(child.attach))) {
         // attach and attachObject don't have an order anyway, so just append
-        added = true
```

Comment:
`added` is only interesting for the `if` condition at Line 389. But as the next line will `return` it will never reach that `if` condition